src/components/shared/passphraseRenderer/index.js

```diff
@@ -0,0 +1,231 @@
+import React from 'react';
+import fillWordsList from 'bitcore-mnemonic/lib/words/english';
+import grid from 'flexboxgrid/dist/flexboxgrid.css';
+import { withTranslation } from 'react-i18next';
+import styles from './passphraseRenderer.css';
+import { PrimaryButton, TertiaryButton } from '../../toolbox/buttons/button';
+
+class PassphraseRenderer extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      indexes: [2, 9],
+      fieldSelected: undefined,
+      displayOptions: undefined,
```

Comment:
🐛 (or 📚?)  Based on the designs, I think the initial state should be the first missing word selected. There is no screen in the designs where both missing words are empty, none of them is selected, and no options are displayed. In that case `state.displayOptions` is not needed, because it should depend on `state.fieldSelected`

This also applies to the previous Story https://github.com/LiskHQ/lisk-hub/pull/2556

Reply:
You're right. I made the changes. It should also be possible to reselect an option once you've already selected it so I changed that also.